src/components/common/button/button.scss

```diff
@@ -0,0 +1,35 @@
+@import '../styles/palette.scss';
+
+.button {
+  width: 300px;
```

Comment:
button sizes cannot be in the generic styles, they must be in the component-style file that creates the button!